modules/cli/src/main/scala/scala/cli/commands/SharedDependencyOptions.scala

```diff
@@ -15,7 +15,10 @@ final case class SharedDependencyOptions(
   @HelpMessage("")
   @Name("repo")
   @Name("r")
-    repository: List[String] = Nil
+    repository: List[String] = Nil,
+  @Group("Scala")
+  @HelpMessage("Add compiler plugins dependencies")
```

Comment:
Maybe we could add `"plugin"` and `"P"` aliases (via `@Name` annotations), which are more convenient to type than `--compiler-plugin`.